packages/backend/src/index.ts

```diff
@@ -55,6 +55,11 @@ export type { VerifyTokenOptions } from './tokens/verify';
  */
 export type { WebhookEvent, WebhookEventType } from './api/resources';
 
+/**
+ * JSON types
+ */
+export type * from './api/resources/JSON';
```

Comment:
🔧  Let's export ONLY the types we want to and avoid `*` as it may result in unwanted exports in the future since the `api/resources/JSON` exports are considered to be consumed internally.

Reply:
@desiprisg PTAL at this one as we'd like to release today

Reply:
Done